packages/drivers/odsp-driver/src/createNewUtils.ts

```diff
@@ -5,21 +5,25 @@
 
 import { v4 as uuid } from "uuid";
 import { ISummaryTree, SummaryType } from "@fluidframework/protocol-definitions";
+import { getDocAttributesFromProtocolSummary } from "@fluidframework/driver-utils";
 import { Uint8ArrayToString, unreachableCase } from "@fluidframework/common-utils";
 import { IOdspSnapshotBlob, IOdspSnapshot, IOdspSnapshotTreeEntry } from "./contracts";
 
 /**
  * Converts a summary(ISummaryTree) taken in detached container to IOdspSnapshot tree
  */
 export function convertSummaryTreeToIOdspSnapshot(summary: ISummaryTree): IOdspSnapshot {
+    const protocolSummary = summary.tree[".protocol"] as ISummaryTree;
+    const documentAttributes = getDocAttributesFromProtocolSummary(protocolSummary);
+    const sequenceNumber = documentAttributes.sequenceNumber;
```

Comment:
This looks right, but glancing at existing usage of this function in UTs, only the .app sub-tree is created there, so I believe you will get undefined for protocolSummary. Which would mean we need to update UTs to be more real, i.e. include both .app part and .protocol sub-trees. @jatgarg - can you please take a look if I got it right?

Reply:
Yes. We can add protocol summary to the test file test. @chensixx Look at createSummary() in odspCreateContainer.spec.ts file and we want to test that summary in our tests in createNewUtilsTests.spec.ts file. Create new summary is unique in that it includes protocol summary with the app summary. 
Also we can change the api name "convertSummaryTreeToIOdspSnapshot" to "convertCreateNewSummaryTreeToIOdspSnapshot".